webapp/apps/dynamic/models.py

```diff
@@ -87,6 +87,25 @@ class DynamicBehaviorSaveInputs(models.Model):
     micro_sim = models.ForeignKey(OutputUrl, blank=True, null=True,
                                   on_delete=models.SET_NULL)
 
+    def get_tax_result(self):
+        """
+        If taxcalc version is greater than or equal to 0.13.0, return table
+        If taxcalc version is less than 0.13.0, then rename keys to new names
+        and then return table
+        """
+        outputurl = OutputUrl.objects.get(unique_inputs__pk=self.pk)
+        taxcalc_vers = outputurl.taxcalc_vers
+        taxcalc_vers = taxcalc_vers.split('.')
+        # older PB versions stored commit reference too
+        # e.g. taxcalc_vers = "0.9.0.d79abf"
+        if len(taxcalc_vers) >=3:
+            taxcalc_vers = taxcalc_vers[:3]
+        taxcalc_vers = tuple(map(int, taxcalc_vers))
+        if taxcalc_vers >= (0, 13, 0):
+            return self.tax_result
+        else:
+            return rename_keys(self.tax_result, PRE_TC_0130_RES_MAP)
+
```

Comment:
@codekansas I would prefer to not copy this method from the taxbrain models code. Do you have any ideas for a cleaner solution?

Reply:
You'll probably want to put this in a `utils.py` file in the parent directory. Hmm... If you do this, you'll probably also want to include

```
from __future__ import absolute_import
```

in the top of the file, so that importing is consistent.

Reply:
@codekansas thanks for the advice. That makes sense.

Another solution would be to create an abstract model class with this method. Then, the `TaxSaveInputs` and `DynamicSaveInputs` classes could inherit this class. It is likely that we will have to add a `get_fields` method in a separate PR that resolves #811. This would be like the base class approach described [here](https://medium.com/eshares-blog/supercharging-django-productivity-at-eshares-8dbf9042825e). An example that is similar to our situation is described [here](http://blog.kevinastone.com/django-model-behaviors.html).